modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -0,0 +1,169 @@
+:_content-type: CONCEPT
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-examples-of-common-use-cases_{context}"]
+= Examples of common use cases
+
+Both Jenkins and Tekton offer capabilities for common CI/CD use cases, such as: 
+
+* Compiling, building images, and deploying images
+* Extending the core capabilities by using plugins
+* Reusing shareable libraries and custom scripts
+
+This section briefly describes the comparable workflows for some common use cases in Jenkins and Tekton.
+
+== Running a maven pipeline in Jenkins and Tekton
+
+Maven is used extensively in the Jenkins workflows. You can map your existing workflow to Tekton by using the following examples:
+
+.Example: Compile, build an image, and deploy to OpenShift using maven on Jenkins
+[source,groovy]
+----
+#!/usr/bin/groovy
+node('maven') {
+    stage 'Checkout'
+    checkout scm
+ 
+    stage 'Build'
+    sh 'cd helloworld && mvn clean'
+    sh 'cd helloworld && mvn compile'
+ 
+    stage 'Run Unit Tests'
+    sh 'cd helloworld && mvn test'
+ 
+    stage 'Package'
+    sh 'cd helloworld && mvn package'
+ 
+    stage 'Archive artifact'
+    sh 'mkdir -p artifacts/deployments && cp helloworld/target/*.war artifacts/deployments'
+    archive 'helloworld/target/*.war'
+ 
+    stage 'Create Image'
+    sh 'oc login https://kubernetes.default -u admin -p admin --insecure-skip-tls-verify=true'
+    sh 'oc new-project helloworldproject'
+    sh 'oc project helloworldproject'
+    sh 'oc process -f helloworld/jboss-eap70-binary-build.json | oc create -f -'
+    sh 'oc start-build eap-helloworld-app --from-dir=artifacts/'
+ 
+    stage 'Deploy'
+    sh 'oc new-app helloworld/jboss-eap70-deploy.json' }
+
+----
+
+.Example: Compile, build an image, and deploy to OpenShift using maven on Tekton
```

Comment:
Similar suggestion as I made above. Consider rephrasing to: `Example: Compile and build an image and deploy it to OpenShift using maven in Tekton.`